SignalR.sln

```diff
@@ -68,6 +68,19 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common", "Common", "{6CEC3D
 		test\Common\XUnitLoggerProvider.cs = test\Common\XUnitLoggerProvider.cs
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "client-ts", "client-ts", "{3A76C5A2-79ED-49BC-8BDC-6A3A766FFA1B}"
+	ProjectSection(SolutionItems) = preProject
+		client-ts\package.json = client-ts\package.json
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.AspNetCore.SignalR.Client.TS.Tests", "Microsoft.AspNetCore.SignalR.Client.TS.Tests", "{ABEEBCD8-4B81-4384-A7DB-F760AD395FFE}"
+	ProjectSection(SolutionItems) = preProject
+		client-ts\Microsoft.AspNetCore.SignalR.Client.TS.Tests\Connection.spec.ts = client-ts\Microsoft.AspNetCore.SignalR.Client.TS.Tests\Connection.spec.ts
```

Comment:
Adding these is not a good idea - VS is trying to compile them and shows errors in the error pane. I will remove.